models/classes/Platform/Repository/Lti1p3RegistrationRepository.php

```diff
@@ -132,8 +131,8 @@ private function getDefaultPlatform(): Platform
         return new Platform(
```

Comment:
We need to update the Tool constructor with new values:

```
https://github.com/oat-sa/extension-tao-lti/pull/270/files#diff-bc1d66e989406291d08866f101d6d838R121-R122
```

It is using the provider ID twice.